README.md

```diff
@@ -5,8 +5,8 @@ Consist of 2 parts:
 + [Telegram bot to play quests.](bot/README.md)
 + [Service for quest creating including back-end and front-end.](website/README.md)
 
-To play quests user must use telegram bot [@QuestMakerBot](https://t.me/QuestMakerBot).
-In bot player need to choose quest and then quest session starts.
+There is telegram bot [@QuestMakerBot](https://t.me/QuestMakerBot) to play quests.
+In bot player is able to choose quest and then quest session starts.
```

Comment:
Maybe "User can choose quest and start to complete it using our telagram bot [@QuestMakerBot](https://t.me/QuestMakerBot)" instead of 8 and 9 lines sounds better.

Reply:
No